xds/internal/client/xds.go

```diff
@@ -620,3 +728,27 @@ func parseEDSRespProto(m *v3endpointpb.ClusterLoadAssignment) (EndpointsUpdate,
 	}
 	return ret, nil
 }
+
+func combineErrors(rType string, topLevelErrors []error, perResourceErrors map[string]error) error {
+	var errStrB strings.Builder
+	errStrB.WriteString(fmt.Sprintf("error parsing %q response: ", rType))
```

Comment:
I think I suggested in the earlier PR that the error might be more readable if newlines were added, and you agreed to it. Could you please do that here. Thanks.